### PR TITLE
docs: add Sequential Tasks Agent documentation and sidebar entry

### PR DESCRIPTION
**Pull request by mendonk**
This pull request includes documentation updates to introduce a new starter project for creating a Sequential Tasks Agent. The most important changes include adding a new markdown file for the Sequential Tasks Agent and updating the sidebar configuration to include this new project.

Documentation updates:

* [`docs/docs/Starter-Projects/starter-projects-sequential-agent.md`](diffhunk://#diff-707ad0bb7d64dc5c6ce9c46c3539b4c7a8aa862b78fb6a4f0899004cd9634829R1-R53): Added new documentation for building a Sequential Tasks Agent, including prerequisites, steps to create and run the flow, and next steps.

Sidebar configuration:

* [`docs/sidebars.js`](diffhunk://#diff-e8aa23ec31a264f87ba92ce3eef2133d44d0190ec08ac6d6eba4913d031294e9R22): Updated the sidebar configuration to include the new Sequential Tasks Agent project.

### CODE REVIEW

File: docs/docs/Starter-Projects/starter-projects-sequential-agent.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+---
+
+Build a **Sequential Tasks Agent** flow for a multi-agent application using multiple **Agent** components.
+
+Each agent has an LLM model and a unique set of tools at its disposal, with **Prompt** components connected to the **Agent Instructions** fields to control the agent's behavior. For example, the **Researcher Agent** has a **Tavily AI Search** component connected as a tool. The **Prompt** instructs the agent how to answer your query, format the response, and pass the query and research results on to the next agent in the flow.
+
+Each successive agent in the flow builds on the work of the previous agent, creating a chain of reasoning for solving complex problems.
+
+## Prerequisites
+- [OpenAI API key created](https://platform.openai.com/)
```
Comment (KimberlyFields) [inline docs/docs/Starter-Projects/starter-projects-sequential-agent.md:14]:
```suggestion
- [An OpenAI API key](https://platform.openai.com/)
```

File: docs/docs/Starter-Projects/starter-projects-sequential-agent.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+- [Tavily AI API key created](https://www.tavily.com/)
```
Comment (KimberlyFields) [inline docs/docs/Starter-Projects/starter-projects-sequential-agent.md:15]:
```suggestion
- [A Tavily AI API key](https://www.tavily.com/)
```

File: docs/docs/Starter-Projects/starter-projects-sequential-agent.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+## Open Langflow and create a new flow
+
+1. Click **New Flow**, and then select **Sequential Tasks Agent**.
+This opens a starter template with the necessary components to run the flow.
+
+![Starter flow for Sequential Tasks Agent](/img/starter-flow-sequential-agent.png)
+
+The Sequential Tasks Agent flow consists of these components:
+
+* The **Agent** components use the connected LLM to reason through the user's input and select among the connected tools to complete its task.
```
Comment (KimberlyFields) [inline docs/docs/Starter-Projects/starter-projects-sequential-agent.md:26]:
```suggestion
* The **Agent** components use the connected LLM to analyze the user's input and select among the connected tools to complete the tasks.
```

File: docs/docs/Starter-Projects/starter-projects-sequential-agent.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+* The **Chat Input** component accepts user input to the chat.
+* The **Prompt** component combines the user input with a user-defined prompt.
+* The **Chat Output** component prints the flow's output to the chat.
+* The **YFinance** tool component provides access to financial data from Yahoo Finance.
+* The **Tavily AI Search** tool component performs AI-powered web searches.
+* The **Calculator** tool component performs mathematical calculations.
+
+## Run the Sequential Tasks Agent flow
+
+1. Add your OpenAI API key to the **Agent** components.
```
Comment (KimberlyFields) [inline docs/docs/Starter-Projects/starter-projects-sequential-agent.md:36]: Is "components" correct or should it be "component"?

**Review by KimberlyFields — APPROVED** [review]
I made a few suggestions, but I'm approving.

File: docs/docs/Starter-Projects/starter-projects-sequential-agent.md (hunk 4 above)
Comment (mendonk) [reply docs/docs/Starter-Projects/starter-projects-sequential-agent.md:36]: There are 3 in the application so it's "components"

**Pull request merged**